views/result.erb

```diff
@@ -0,0 +1,9 @@
+<% if @throw == @computer_option %>
+<h2>Draw.</h2>
+<% elsif @throw == "Rock" && @computer_option == "Scissors" || @throw == "Scissors" && @computer_option == "Paper" || @throw == "Paper" && @computer_option == "Rock" %>
```

Comment:
All that logic, it is best practice to either keep it in controllers (if these are light tasks), or even better, if you would make your class keep it and execute it.